services/settings.py

```diff
@@ -1,8 +1,5 @@
 import os
 
-from django.utils import importlib
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'settings_local')
 
-
-# get the right settings module
-settings = importlib.import_module(
-    os.environ.get('DJANGO_SETTINGS_MODULE', 'settings_local'))
+from django.conf import settings  # noqa isort:nofix
```

Comment:
I know that `services` tries hard to not start the django engine but eventually it happens anyway so I hope this doesn't pose any problems. Needs explicit testing though

Reply:
I always hated this behavior but at the same time changing it is scary - to test it, we need to perform some load testing, make sure there are no perf regressions here as the endpoint is called *a lot*. I don't think we're equipped for that atm...

Reply:
Note: Also worth checking that settings are imported correctly too.

Reply:
I got all 3 services started locally and got reasonable results from my request-tests without knowing what I'm doing obviously. I'll ask ops regarding regression tests but I don't think this is a merge-blocker since there are quite a few other imports in the `services` package that import `django.conf.settings` too.